### PR TITLE
Add OS parameter into the ad request

### PR DESCRIPTION
**Pull request by myunggeun-moloco**
This PR aims to add an OS parameter to the ad request as it is required in the adservfnt.

### CODE REVIEW

File: publisher/moloco.js [hunk 1]
```diff
@@ -27,24 +27,25 @@ function constructNativeAdDiv(nativeAd, nativeAdDiv) {
...
+  this.extra = encodeURIComponent(data["extra"] || "");
+  this.os = data["os"];
 }
 
 // Send ad request to Moloco.
-MolocoSDK.prototype.requestAd = function(adDiv) {
+MolocoSDK.prototype.requestAd = function (adDiv) {
   const orientation = window.innerWidth > window.innerHeight ? "l" : "p";
-  let url = this.endpoint + "&id=" + this.adUnit + "&udid=ifa:" + this.idfa + "&bundle=" + this.bundle + "&iso=" + this.country + "&w=" + this.width + "&h=" + this.height + "&o=" + orientation + "&ufid=" + uuidv4() + "&x=" + this.extra;
+  let url = this.endpoint + "&id=" + this.adUnit + "&udid=ifa:" + this.idfa + "&bundle=" + this.bundle + "&iso=" + this.country + "&w=" + this.width + "&h=" + this.height + "&o=" + orientation + "&ufid=" + uuidv4() + "&x=" + this.extra + "os=" + this.os;
```
Comment (soohan-ahn) [inline publisher/moloco.js]: [MUST]
I think you need to add `&` in front of "os="

**Review by soohan-ahn — CHANGES_REQUESTED** [review]
Thank you for the implementation.
I left a comment.

File: publisher/moloco.js (hunk 1 above)
Comment (myunggeun-moloco) [reply publisher/moloco.js]: Yup, I updated the code with checking the os parameter is undefined.

**Review by soohan-ahn — APPROVED** [review]
LGTM!

**Comment by hyeonseo** [comment]
Looks good to me as well

**Comment by myunggeun-moloco** [comment]
Thanks for the review!

**Pull request merged**